bevy_rapier3d/examples/multi_world3.rs

```diff
@@ -0,0 +1,120 @@
+use bevy::{input::common_conditions::input_just_pressed, prelude::*};
+use bevy_rapier3d::prelude::*;
+
+const N_WORLDS: usize = 2;
+
+fn main() {
+    App::new()
+        .insert_resource(ClearColor(Color::srgb(
+            0xF9 as f32 / 255.0,
+            0xF9 as f32 / 255.0,
+            0xFF as f32 / 255.0,
+        )))
+        .add_plugins((
+            DefaultPlugins,
+            RapierPhysicsPlugin::<NoUserData>::default()
+                .with_default_world(RapierContextInitialization::NoAutomaticRapierContext),
+            RapierDebugRenderPlugin::default(),
+        ))
+        .add_systems(
+            Startup,
+            ((create_worlds, setup_physics).chain(), setup_graphics),
+        )
+        .add_systems(Update, move_platforms)
+        .add_systems(
+            Update,
+            change_world.run_if(input_just_pressed(KeyCode::KeyC)),
+        )
+        .run();
+}
+
+fn create_worlds(mut commands: Commands) {
+    for i in 0..N_WORLDS {
+        let mut world = commands.spawn((RapierContext::default(), WorldId(i)));
```

Comment:
`WorldId` is only used in the context of this example, I'm not sure this would add too much value in real code, as it's only used for hardcoding position offsets and platform movement. I'd argue it's not a great practice to enforce index for worlds, the "correct" index is the `Entity` itself. 

A user is still free to add its own index (like the example), or use uuid, or names or anything else, I just feel it's a tiny bit too opinionated for the official API.

We could argue a bundle would be interesting to inject `RapierConfiguration` and/or `SimulationToRenderTime` at the same time, but `RapierConfiguration` is initialized depending on `RenderContext` so I'm not too confident with it.

This initialization will probably change with https://github.com/dimforge/bevy_rapier/issues/502, where we'll have more reason to introduce a bundle.

Reply:
I was thinking ahead to #502 when mentioning the bundle idea, but it makes sense not to add a bundle when there is only one component at the moment.